src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -846,14 +895,14 @@ private void ApplyRename(Solution newSolution, IUIThreadOperationContext operati
                 var finalChanges = _workspace.CurrentSolution.GetChanges(_baseSolution);
 
                 var finalChangedIds = finalChanges
-                        .GetProjectChanges()
-                        .SelectMany(c => c.GetChangedDocuments().Concat(c.GetAddedDocuments()))
-                        .ToList();
+                    .GetProjectChanges()
+                    .SelectMany(c => c.GetChangedDocuments().Concat(c.GetAddedDocuments()))
+                    .ToList();
 
+                await _threadingContext.JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
```

Comment:
we already switched above, do we have to switch here again?

Reply:
totally true.  this is not needed. t hanks!